providers/edge/src/airflow/providers/edge/plugins/edge_executor_plugin.py

```diff
@@ -36,8 +36,7 @@
 from airflow.utils.yaml import safe_load
 from airflow.www import utils as wwwutils
 from airflow.www.auth import has_access_view
-from airflow.www.constants import SWAGGER_BUNDLE, SWAGGER_ENABLED
-from airflow.www.extensions.init_views import _CustomErrorRequestBodyValidator, _LazyResolver
+from airflow.providers.fab.www.extensions.init_views import _CustomErrorRequestBodyValidator, _LazyResolver
```

Comment:
These imports should just move, unchanged, into `_get_airflow_2_api_endpoint` instead. That's already only run on AF2.

Reply:
Need to double check that this does not break compatability with Airflow 2 - I am not sure whether plugin is loaded in providers test. Once CI is green let me check this manually please.

Reply:
Moved.